resources/views/user/jobs/failed_index.blade.php

```diff
@@ -8,51 +8,50 @@
         <div class="card-body px-0 table-responsive">
             <table class="table table-striped mb-0">
                 <thead>
-                <tr>
-                    <th>@lang('ID')</th>
-                    <th>@lang('Verbindung')</th>
-                    <th>@lang('Queue')</th>
-                    <th>@lang('Payload') / @lang('Exception')</th>
-                    <th>@lang('Datum')</th>
-                </tr>
+                    <tr>
+                        <th>@lang('ID')</th>
+                        <th>@lang('Verbindung')</th>
+                        <th>@lang('Stichwort')</th>
+                        <th>@lang('Nutzlast') / @lang('Ausnahme')</th>
```

Comment:
These should be left as `Queue`, `Payload` and `Exception` :)
They are commonly used terms in programming